tests/bearlib/aspects/CollectionsTest.py

```diff
@@ -9,10 +9,18 @@
 class aspectlistTest:
 
     def test__init__(self):
-        with pytest.raises(aspectTypeError) as exc:
+        list_of_aspect = aspectlist(['CommitMessage.Shortlog',
+                                     'CommitMessage.Body'])
+        mix_of_aspect = aspectlist(['CommitMessage.Shortlog',
+                                    Metadata.CommitMessage.Body])
+        assert isinstance(list_of_aspect, aspectlist)
+        assert list_of_aspect[0] is Metadata.CommitMessage.Shortlog
+        assert list_of_aspect[1] is Metadata.CommitMessage.Body
```

Comment:
This part of commit is already implemented in other PR and will be removed in next iteration